sklearn/linear_model/_omp.py

```diff
@@ -646,11 +650,32 @@ def fit(self, X, y):
         self : object
             returns an instance of self.
         """
+        if self.normalize == 'deprecate':
+            _normalize = True
+        else:
+            _normalize = self.normalize
+
+        if not _normalize:
+            warnings.warn(
+                "'normalize' was deprecated in version 0.24 and will be"
+                " removed in 0.26.", FutureWarning
+            )
+        else:
+            warnings.warn(
+                "'normalize' was deprecated in version 0.24 and will be"
+                " removed in 0.26. If you wish to keep an equivalent"
+                " behaviour, use  Pipeline with a StandardScaler in a"
+                " preprocessing stage:"
+                "  model = make_pipeline( \n"
```

Comment:
I'm not sure that using `\n` will make the rendering better (we don't know our users screen width)